### PR TITLE
Roll back MKL related changes

### PR DESCRIPTION
**Pull request by steventk-g**

### CODE REVIEW

**Review by JackCaoG — APPROVED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -47,7 +47,7 @@ RUN if [ "${git_clone}" = "true" ]; then github_branch="${release_version}" && \
   rm -rf xla && \
   git clone -b "${github_branch}" --recursive https://github.com/pytorch/xla && \
   cd / && \
-  git clone -b "${github_branch}" --recursive https://github.com/pytorch-tpu/examples tpu-examples; fi
+  git clone -b "master" --recursive https://github.com/pytorch-tpu/examples tpu-examples; fi
```
Comment (JackCaoG) [inline docker/Dockerfile:50]: let's revert this change

Comment (steventk-g) [reply docker/Dockerfile]: Done
